layouts/blog/single.html

```diff
@@ -1,8 +1,8 @@
-{{ define "title" }}
-The gRPC Blog &ndash; {{ .Title }}
-{{ end }}
+{{ define "title" -}}
+{{ .Title }} &ndash; Blog | {{ site.Title -}}
```

Comment:
Note the little variance in the title for blog entries: "page-title - Blog | gRPC"

Reply:
I've removed "Blog"